### ISSUE TITLE
Error 500 when opening website

### ISSUE THREAD

**Issue by dsv591** [nuxsmin/sysPass#256]
Hello

I'm trying to install SysPass over Wamp in Windows. I copied the files to the server and launched the website on the browser but I get an error 500.

I have enabled the mod_rewrite (which I don't know if it's needed but it comes disabled by default), still get the same error message.

How can I get the details of the error?

Thank you

**Comment by dsv591** [comment]
anyone? :(

**Comment by nuxsmin** [comment]
Hello,

can you take a look to the Apache error log?. sysPass could throw an exception or an error, but the 500 code is likely to a misconfigured webserver.

**Comment by nuxsmin** [comment]
Mod_rewrite is not needed.

You can found some install info at wiki.syspass.org

**Comment by dsv591** [comment]
Just installed Wamp and copied the files, the installer gives the error so haven't actually set anything up.

Error log is empty, access log shows this
10.253.34.10 - - [24/Aug/2016:16:12:49 -0400] "GET /syspass/ HTTP/1.1" 500 -

But it isn't very helpful :/

**Comment by dsv591** [comment]
Ok, I got the apache error log working:

[Wed Aug 24 16:52:52.340977 2016] [:error] [pid 3288:tid 944] [client 10.253.34.10:58217] PHP Stack trace:
[Wed Aug 24 16:52:52.340977 2016] [:error] [pid 3288:tid 944] [client 10.253.34.10:58217] PHP   1. {main}() C:\\wamp\\www\\syspass\\index.php:0
[Wed Aug 24 16:52:52.340977 2016] [:error] [pid 3288:tid 944] [client 10.253.34.10:58217] PHP   2. require() C:\\wamp\\www\\syspass\\index.php:28
[Wed Aug 24 16:52:52.340977 2016] [:error] [pid 3288:tid 944] [client 10.253.34.10:58217] PHP   3. SP\\Init::start() C:\\wamp\\www\\syspass\\inc\\Base.php:44
[Wed Aug 24 16:52:52.340977 2016] [:error] [pid 3288:tid 944] [client 10.253.34.10:58217] PHP   4. SP\\Language::setLanguage() C:\\wamp\\www\\syspass\\inc\\Init.class.php:119
[Wed Aug 24 16:52:52.340977 2016] [:error] [pid 3288:tid 944] [client 10.253.34.10:58217] PHP   5. SP\\Language::setLocales() C:\\wamp\\www\\syspass\\inc\\Language.class.php:70
[Wed Aug 24 16:52:52.340977 2016] [:error] [pid 3288:tid 944] [client 10.253.34.10:58217] PHP   6. setlocale() C:\\wamp\\www\\syspass\\inc\\Language.class.php:136
[Wed Aug 24 16:52:52.356576 2016] [:error] [pid 3288:tid 944] [client 10.253.34.10:58217] PHP Fatal error:  Call to undefined function SP\\bindtextdomain() in C:\\wamp\\www\\syspass\\inc\\Language.class.php on line 138
[Wed Aug 24 16:52:52.356576 2016] [:error] [pid 3288:tid 944] [client 10.253.34.10:58217] PHP Stack trace:
[Wed Aug 24 16:52:52.356576 2016] [:error] [pid 3288:tid 944] [client 10.253.34.10:58217] PHP   1. {main}() C:\\wamp\\www\\syspass\\index.php:0
[Wed Aug 24 16:52:52.356576 2016] [:error] [pid 3288:tid 944] [client 10.253.34.10:58217] PHP   2. require() C:\\wamp\\www\\syspass\\index.php:28
[Wed Aug 24 16:52:52.356576 2016] [:error] [pid 3288:tid 944] [client 10.253.34.10:58217] PHP   3. SP\\Init::start() C:\\wamp\\www\\syspass\\inc\\Base.php:44
[Wed Aug 24 16:52:52.356576 2016] [:error] [pid 3288:tid 944] [client 10.253.34.10:58217] PHP   4. SP\\Language::setLanguage() C:\\wamp\\www\\syspass\\inc\\Init.class.php:119
[Wed Aug 24 16:52:52.356576 2016] [:error] [pid 3288:tid 944] [client 10.253.34.10:58217] PHP   5. SP\\Language::setLocales() C:\\wamp\\www\\syspass\\inc\\Language.class.php:70

**Comment by marnylopez** [comment]
You should activate extension=php_gettext.dll in php.ini

**Comment by dsv591** [comment]
Got it, I was missing the gettext extension

**Comment by rahra** [comment]
I suggest to update the prerequisites in the installation manuals that the PHP module `gettext` is a requirement.

**Comment by nuxsmin** [comment]
Hello , it shouldn't be needed that  module since sysPass relies in native GNU gettext tools, that are bundled within almost every OS, but it seems that  Windows installations doesn't.

Regards

**Comment by rahra** [comment]
I got the following error message in the Apache error log and syspass didn't work:
`PHP Fatal error:  Call to undefined function SP\\Core\\bindtextdomain() in /usr/local/www/apache24/sysPass-2.1.15.17101701/inc/SP/Core/Language.class.php on line 161`

The message disappeared and syspass worked immediately after I installed `php56-gettext` module.

I'm running it on Apache 2.4.33 with mysql 5.6.39 and PHP 5.6.34 on FreeBSD 10.4. All packages are up to date.
